patches/aesthetics.py

```diff
@@ -261,7 +261,53 @@ def reduceMessageLengths(rom, rnd):
         "#####, how do I download RAM?",
         "Is this a delayed April Fool's Joke?",
         "Dayoman is close to getting 120% in 1:20!",
-        "Play as if your footage will go in a\nSummoning Salt video."
+        "Play as if your footage will go in a\nSummoning Salt video.",
+        "I hope you prepared for our date later.",
+        "Isn't this the game where you date a seagull?",
+        "!tele broken, pls fix",
+        "You look pretty good for a guy who probably drowned.",
+        "I heard Matty's a cheater.",
+        "Remember, we race on Sundays.",
+        "Pink Batman delayed this race for lunch.",
+        "This randomizer was made possible by players like you. \n \n Thank you!",
+        "Now with real fake doors!",
+        "You could be doing something productive right now.",
+        "No eggs were harmed in the making of this game.",
+        "I'm helping the goat, \ncatfishing Mr. Write is kinda the goal.",
+        "There are actually two LADX randomizers.",
+        "Did you actually dump this cart?"
+        "You're not gonna cheat... \n ...right?",
+        "Mamu's singing is so bad it wakes the dead.",
+        "Don't forget the Richard picture.",
+        "Are you sure you wanna do this?  I kinda like this island.",
+        "SJ, BT, WW, OoB, HIJKLMNOP.",
+        "$5 in the swear jar.  Now.",

```

Comment:
Is there a symbol for `$`?

Reply:
Good point, I'll edit it when I get back home

Reply:
Adjusted a few lines that had symbols that probably aren't in the LADX font.